1 table.sql

```diff
@@ -0,0 +1,30 @@
+use MusicNoteDB 
+go 
+
+drop table if exists MusicRental
+go 
+create table dbo.MusicRental(
+    InstrumentId int not null identity primary key,  
+    CustomerFirstName varchar(30) not null constraint c_MusicNote_CustomerFirstName_cannot_be_blank check (CustomerFirstName > ''),
+    CustomerLastName varchar(30) not null constraint c_MusicNote_CustomerLastName_cannot_be_blank check (CustomerLastName > ''),
+    Instrument varchar(20) not null constraint c_MusicNote_Instrument_cannot_be_blank check (Instrument > ''),
```

Comment:
Nice that you made this constraint, but the owner did not say he is limited to these 4 categories, happens to be those are the only ones in his data so far. Rather add constraint that can't be blank.